package.json

```diff
@@ -499,10 +499,15 @@
                     "when": "view == dockerRegistries && viewItem =~ /commontag/i && !(viewItem =~ /commontag;.*dockerhub/i)",
                     "group": "regs_tag_1_general@3"
                 },
+                {
+                    "command": "vscode-docker.registries.inspectRemoteImageManifest",
+                    "when": "view == dockerRegistries && viewItem =~ /commontag/i && !(viewItem =~ /commontag;.*dockerhub/i)",
```

Comment:
This is not working for GitHub. Either needs to be disabled for GitHub or fixed.

Reply:
Nevermind...weird...